app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Lists of user </h1>
\ No newline at end of file
+<ul>
+<% @users.each do |user| %>
+<li><%=user.name%><span>Number of 
+Posts: <%=user.posts_counter%></span></li>
```

Comment:
- [ ] @Lul-Abdifan It would be nice if you added the user info as a link, this will make it easier to navigate from the users page to a particular user page(show page) (imagine the user doesn't know what routes are and have to rely on clicking a link to access the next page)